index.js

```diff
@@ -24,67 +26,81 @@ function getRandomNumber(min, max) {
 
 function checkGuess() {
   // Get value from guess input element
-  const guess = parseInt(guessInput.value, 10);
+  let guess = parseInt(guessInput.value, 10);
   attempts = attempts + 1;
 
+  if (guess < 1) {
+    // Display a message for guesses lower than 0
+    numberLessThanZero.style.display = "";
+    return;
+  }
+
+  if (guess > 99) {
+    // Display a message for guesses higher than 99
+    numberHigherThan99.style.display = "";
+    return; // Exit the function to prevent further execution
+  }
+
   hideAllMessages();
 
   if (guess === targetNumber) {
-    numberOfGuessesMessage.style.display = '';
+    numberOfGuessesMessage.style.display = "";
```

Comment:
When you're working on larger projects, it will be important to make sure your PRs just do one thing. This is an innocuous formatting change, but it adds to the burden on the reviewer who just wants to see the bug fixes. On a real project, having a separate PR for formatting changes would be good.